ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -290,11 +299,13 @@ module.exports = Marionette.LayoutView.extend({
     let value = Common.duplicate(this.model.get('value'))
     const currentComparator = this.model.get('comparator')
     value = this.transformValue(value, currentComparator)
-    const propertyJSON = generatePropertyJSON(
-      value,
-      this.model.get('type'),
-      currentComparator
-    )
+    let type = this.model.get('type')
+
+    if (currentComparator === 'IS EMPTY') {
```

Comment:
Can we update `generatePropertyJSON` instead of modifying the value/type here?

Reply:
That was what I tried originally, however there were some issues caused with the value parameter that I ran into, so it was instead easier to do it this way.